Picard_inputs/KMT2A_exon_processing.R

```diff
@@ -0,0 +1,13 @@
+setwd("~/new_projects/tandemhunter_coverage")
```

Comment:
Is setwd necessary here? Not sure a folder named 'new_projects' is being used in this app - though I may have overlooked it. 
Nitpick: some <- arrows further down need spaces adding in front of them